src/libs/sfdk/vboxvirtualmachine.cpp

```diff
@@ -429,6 +428,21 @@ void VBoxVirtualMachinePrivate::setVideoMode(const QSize &size, int depth,
         setExtraData(key, value, Sdk::instance(), [=](bool ok) { *allOk &= ok; });
     };
 
+    {
+        BatchComposer composer = BatchComposer::createBatch("setLastNormalWindowPosition");
+        fetchExtraData("GUI/LastNormalWindowPosition", Sdk::instance(), [=, batch = composer.batch()](QString data, bool ok) {
+            if (!ok)
+                return;
+            QTC_ASSERT(data.startsWith("Value:"), return);
+            data = data.section(':',1,1).trimmed();
+            QStringList splitData = data.split(',');
```

Comment:
I would prefer joining this and making the three variables const (and ref in case of `data`).

Reply:
fixed